web/src/components/profile/ProfileTabHeader.tsx

```diff
@@ -24,8 +24,8 @@ export const ProfileTabHeader = (props: Props): ReactElement => {
   };
 
   return (
-    <div data-testid="profile-header">
-      <hr className="margin-top-4 margin-bottom-2" aria-hidden={true} />
+    <div data-testid="profile-tab-header">
+      <hr className="desktop:margin-top-0 margin-top-4 margin-bottom-2" aria-hidden={true} />
```

Comment:
[dust] `<hr>` has specific meaning with accessibility. Do we want an `<hr>` here or do we want a line, like a border? Is there a thematic or section break?

Reply:
Would love to learn what you mean exactly. 

This is currently aria-hidden

Reply:
Follow up ticket